### PR TITLE
Basic Sprint completed

### PR DESCRIPTION
**Pull request by metten0**

### CODE REVIEW

**Review by BlackBelt2817 — COMMENTED** [review]
Nice job Jason. You were very thorough with your questions and the 1d looks good. Keep up the hard work!

File: security/PracticalSecurity.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+1.  Do something to improve your personal security setup -
+
+_Answer_
+
+Two Factor Auth: I set up 2 factor authentication for my Amazon account. They have been bugging me to do it for a while now, so I figured I would take care of both my Amazon security needs and my Sprint Challenge needs at the same time. The setup was pretty easy and hassle-free I provided my cell phone number as 2nd layer of authentication. When I log in to Amazon, it now sends me a text with a 6-digit code. I enter the code into a form field, and I am granted access to my account. This 2 factor auth protects should my password get stolen while making it more difficult to access the account through other nefarious methods. Now my Amazon account is as secure as my bank account...which still isn't all that great, but it's better.
```
Comment (Velsu) [inline security/PracticalSecurity.md:5]: Two factor authentication is one of the best ways to secure Your account.

File: security/ThreatModel.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+2.  Human Misuse -  
+    Inside or external attackers could potentially gain access to or misuse credentials to harm or otherwise compromise your applications, systems, or networks.
+
+3.  Crimeware -  
+    Crimeware is any software that is installed within applications, systems, or networks through internal (i.e. locally installed) or external (i.e. email attachments, etc) means for nefarious purposes. Recently, the most common form of crimeware is _ransomware_, which is software that removes the user's access to data until such time a ransom is paid.
+
+# Mitigation
+
+A key to mitigating risk is keeping one's attack surface small, and properly managing the interaction between software, hardware, and networks. In terms of software, a small attack surface also provides the added benefit of a clean and efficient UI.
```
Comment (Velsu) [inline security/ThreatModel.md:31]: Great example of threat model and possible mitigations.

**Review by Velsu — COMMENTED** [review]
Great job on the sprint challenge. I love the answers and 1d looks great.

**Pull request closed without merging**